intake_cesm/manage_collections.py

```diff
@@ -0,0 +1,301 @@
+import fnmatch
+import logging
+import os
+import shutil
+import re
+
+from subprocess import Popen, PIPE
+
+import pandas as pd
+import yaml
+from tqdm import tqdm
```

Comment:
`tqdm` needs to be added to the list of requirements in `setup.py` by changing this line:

`install_requires = ["intake", "xarray", "pyyaml"]`